Sources/System/Process/SignalSet.swift

```diff
@@ -0,0 +1,86 @@
+
+extension CInterop {
+  public typealias SigSet = UInt32 // TODO: Linux?
+}
+
+
+public struct SignalSet: Hashable, Codable {
```

Comment:
I wonder if `Codable` conformance is a good idea here -- `sigset_t` values aren't necessarily portable between platforms/architectures.

Reply:
They are definitely not. Neither are signal numbers, errno codes, or so.

Reply:
That's true, they would have to be encoded/decoded as the string of function calls.

Reply:
I don't suppose there is a portable/easy way to list the full contents of a signal set, is there? (If we had one, we could maybe add `Collection` and/or `SetAlgebra` conformance. OTOH, that'd probably be gilding the lily...)